### PR TITLE
readme-lhs added.

### PR DESCRIPTION
**Pull request by tonyday567**

### CODE REVIEW

File: readme-lhs.hsfiles [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+~~~
+stack new projectname readme-lhs
+cd projectname
+<<edit readme.lhs>>
+stack build
+~~~
+
+`stack solve` and `stack build` if anything goes wrong.
```
Comment (mgsloan) [inline readme-lhs.hsfiles]: No such command as `stack solve`.  That is a nice name, though, heh

**Comment by mgsloan** [comment]
We aren't really sure what we want to include in the templates repo or not - https://github.com/commercialhaskell/stack-templates/issues/55

Will you personally find this useful?  It is not a pattern I use or see often.

**Comment by tonyday567** [comment]
Appreciate #55.

The pattern rarity is a main argument for inclusion.  A project with a modern prelude, a literate style, no test-suite, no executable takes hours to create from the simple template.  You get a lock down on technical debt and a `stack init && readme` loop out of the box.

I tested here: https://github.com/tonyday567/readme-lhs.

**Comment by mgsloan** [comment]
I don't think i

**Pull request merged**